hudi-utilities/src/main/java/org/apache/hudi/utilities/config/CloudSourceConfig.java

```diff
@@ -137,4 +147,12 @@ public class CloudSourceConfig extends HoodieConfig {
       .sinceVersion("0.14.1")
       .withDocumentation("specify this value in bytes, to coalesce partitions of source dataset not greater than specified limit");
 
+  public static final ConfigProperty<Integer> MAX_FETCH_TIME_PER_SYNC_MS = ConfigProperty
+      .key(STREAMER_CONFIG_PREFIX + "source.cloud.meta.max.fetch.time.per.sync.ms")
+      .defaultValue(1)
```

Comment:
Is 1 millisecond too small as the default value?

Reply:
agree. it actually should be some relatively large value since it works as a time-out.

Reply:
This should be one minute actually, pushing a fix. 
` public static final long DEFAULT_MAX_TIME_MESSAGES_MILLIS = TimeUnit.MINUTES.toMillis(1);`

Reply:
Fixed it in https://github.com/apache/hudi/pull/10171